src/services/LogFileManager/index.ts

```
@@ -284,7 +287,11 @@
                 logEventNum,
             ] = r;
 
-            messages.push(msg);
+            const printedMsg = (isPrettified) ?
+                `${JsBeautify(msg)}\n` :
```

ESLint

[error] A function with a name starting with an uppercase letter should only be used as a constructor.

(new-cap)